Pages/Cart/styles.css

```diff
@@ -0,0 +1,97 @@
+@import url('/root.css');
```

Comment:
![review1](https://user-images.githubusercontent.com/68228131/154761004-0211dee7-36af-4667-8f64-69195a8f58d7.png)

one css file request is failing.